lighthouse-core/lib/network-recorder.js

```diff
@@ -168,15 +165,17 @@ class NetworkRecorder extends EventEmitter {
   }
 
   /**
-   * Construct network records from a log of devtools protocol messages.
-   * @param {!DevtoolsLog} log
+   * Construct network records from a log of devtools protocol messages. If a
+   * driver is provided, it will be used for request content lookups.
+   * @param {!DevtoolsLog} devtoolsLog
+   * @param {!Driver=} driver
```

Comment:
also, it appears that all the other `request*()` methods on `WebInspector.NetworkRequest` don't send additional protocol commands, they already have what they need locally